oslabs-beta/Kubiquity
# Package electron app

WHAT DOES THIS PR DO:

- Add packaging scripts for linux, mac, and windows

HOW TO TEST THIS PR:

1.  Run `gh pr checkout 46 && npm install && yarn`
2. Run `npm run package-mac` (replace mac with whatever OS you're using. Options are win and linux)
3. Verify that a release-build folder is created and the packaged app is inside.
4. Run the app and verify its working.

ADDITIONAL INFORMATION:

There might be something weird with me packaging the mac app on a Linux distribution on a windows machine. Might be better to package apps in their respective operating systems.
